src/components/Form/Form.js

```diff
@@ -111,6 +111,25 @@ export default function Form({ onAddProduct }) {
 							</ImageWrapper>
 						</ImageContainer>
 					</InputSingleContainer>
+					<InputSingleContainer>
+						<Label htmlFor="name">name</Label>
+						<Input
+							id="name"
+							type="text"
+							aria-invalid={errors.name ? 'true' : 'false'}
+							{...register('name', {
+								required: true,
+								maxLength: 20,
+							})}
+							placeholder="..."
+						/>
+						{errors.name && errors.name.type === 'required' && (
+							<span>please enter your name</span>
+						)}
+						{errors.name && errors.name.type === 'maxLength' && (
+							<span>Please use less than 20 characters</span>
+						)}
```

Comment:
Very cool custom error messages :)
However if more than 20 characters are entered then spacing on mobile view messes up.
- Create button shown on top of last form input
- Upload button moved off-screen
Maybe fix with final styling?

Reply:
I will keep that in min in my next US. It's all about clean up.